Space Wolves.cat

```diff
@@ -1365,35 +1368,11 @@ Head-taker: If this Gambit is selected, the Opposing Player may not claim an Out
           </conditions>
         </modifier>
       </modifiers>
-      <categoryLinks>
```

Comment:
The category links for force slots should be on the entry links not on the roots, per the implementation guide.